worldeditadditions/lib/selection/cloud.lua

```diff
@@ -0,0 +1,48 @@
+--  ██████ ██       ██████  ██    ██ ██████
+-- ██      ██      ██    ██ ██    ██ ██   ██
+-- ██      ██      ██    ██ ██    ██ ██   ██
+-- ██      ██      ██    ██ ██    ██ ██   ██
+--  ██████ ███████  ██████   ██████  ██████
+worldeditadditions.add_pos = {}
+local wea = worldeditadditions
+function worldeditadditions.add_point(name, pos)
+	if pos ~= nil then
+		-- print("[set_pos1]", name, "("..pos.x..", "..pos.y..", "..pos.z..")")
+		if not worldedit.pos1[name] then worldedit.pos1[name] = vector.new(pos) end
+		if not worldedit.pos2[name] then worldedit.pos2[name] = vector.new(pos) end
+		
+		worldedit.marker_update(name)
+		
+		local volume_before = worldedit.volume(worldedit.pos1[name], worldedit.pos2[name])
+		
+		worldedit.pos1[name], worldedit.pos2[name] = worldeditadditions.vector.expand_region(worldedit.pos1[name], worldedit.pos2[name], pos)
+		
+		local volume_after = worldedit.volume(worldedit.pos1[name], worldedit.pos2[name])
+		
+		local volume_difference = volume_after - volume_before
+		
+		worldedit.marker_update(name)
+		worldedit.player_notify(name, "Expanded region by "..volume_difference.." nodes")
+	else
+		worldedit.player_notify(name, "Error: Too far away (try raising your maxdist with //farwand maxdist <number>)")
+		-- print("[set_pos1]", name, "nil")
+	end
+end
+function worldeditadditions.clear_points(name, pos)
```

Comment:
Again, the name here could use some work. What about `sclear`?

Reply:
again let's just add `.selectors` to the name

Reply:
Sure, that works. But I think `.selection` might be a slightly better name?

Reply:
selection it is then